index.html

```diff
@@ -411,7 +162,7 @@ <h2>About me</h2>
           <img alt="division" src="img/line2.png" class="line2" />
         </section>
         <section class="skills d-flex">
-          <ul>
+          <ul class="tech">
             <li>
               <h2 class="lan">Language</h2>
               <ul class="eniimi">
```

Comment:
- [ ] [REQUIRED] Please note that the list of `Languages`, `frameworks` and Skills` in the `ul` tag are aligned left but yours are aligned to the center. Kindly fix this so that your design can look more like the template provided.
Kindly refer to the images I attached earlier.